xbmc/video/VideoThumbLoader.cpp

```diff
@@ -492,7 +492,9 @@ bool CVideoThumbLoader::LoadItemLookup(CFileItem* pItem)
 
 void CVideoThumbLoader::SetArt(CFileItem &item, const std::map<std::string, std::string> &artwork)
 {
-  item.SetArt(artwork);
+  for (const auto& art : artwork)
```

Comment:
This shoud be replaced by `CGUIListItem::AppendArt` as the implementation is exactly the same.